libs/agno/agno/knowledge/pdf_url.py

```diff
@@ -38,3 +38,93 @@ async def async_document_lists(self) -> AsyncIterator[List[Document]]:
                 yield await self.reader.async_read(url=url)
             else:
                 logger.error(f"Unsupported URL: {url}")
+
+    def load_url(
+        self,
+        url: str,
+        metadata: Optional[Dict[str, Any]] = None,
+        recreate: bool = False,
```

Comment:
We don't need this I think

Reply:
i didn't understand. we dont need to pass these params in `load_url()`?
But we have to pass metadata, url etc right? how else will we pass `metadata` to the knowledge otherwise?